RadarSDK/Include/Radar.h

```diff
@@ -845,27 +845,69 @@ logConversionWithNotification
        completionHandler:(RadarSearchPlacesCompletionHandler)completionHandler NS_SWIFT_NAME(searchPlaces(near:radius:chains:chainMetadata:categories:groups:limit:completionHandler:));
 
 /**
- Gets the device's current location, then searches for geofences near that location, sorted by distance.
+Gets the device's current location, then searches for geofences near that location, sorted by distance.
 
- @param radius The radius to search, in meters. A number between 100 and 10000.
+ @param radius The radius to search, in meters. A number between 100 and 10000. If 0 is entered, the server use its default value.
```

Comment:
```suggestion
 @param radius The radius to search, in meters. A number between 100 and 10000. If -1 is supplied, the search is performed without a radius limit.
```

Reply:
I'm tempted to say that we actually should provide a function sans `radius` (and maybe sans the rest of the optionals) for simplicity's sake. Passing in -1 is a bit weird to get the "default" unlimited behavior

Reply:
Right now the sdk is just passing that int from the user to the server, the server actually does another check to ensure that the radius is more than 100, and sets it to 100 if we do not provide a valid value.

In this case I believe that our SDK can be thought of as the client end stub of a RPC. Since the input verification logic is already handled by the server, we should just pass the value along and pass the warnings back.

However, our SDK does not currently parse the warnings from the server, is this a new pattern we want to introduce?

Reply:
@lmeier I do agree that passing in -1 may be a little weird for default unlimited behavior. But I also think that growing the number of interface a function of 2x each time we add in a new optional can get ugly in the future. I feel we should bite the bullet and introduce NSNumbers into our interface to support optional bools and ints.

Reply:
As it's currently written, `-1` will throw an error if passed. It's either optional, or will support 1 to 10,000.

(we will warn on anything < 100 to preserve backwards compatibility for existing API calls with a small radius)

https://github.com/radarlabs/server/pull/5689/files#diff-6a70e98a5019dec506cc2ac52d75fac8981c3796151166b54e6de676ac5b61cfR365-R369

Reply:
the sdk won't pass the -1 to the server, it will just ignore that parameter